pep-0675.rst

```diff
@@ -141,6 +141,7 @@ from a format string using ``user_id``, and cannot be passed to
     def query_user(conn: Connection, user_id: str) -> User:
         query = f"SELECT * FROM data WHERE user_id = {user_id}"
         conn.execute(query)
+        ...
         # Error: Expected LiteralString, got str.
```

Comment:
```suggestion
        conn.execute(query)  # Error: Expected LiteralString, got str.
        ...
```

The typecheck error occurs on this line, not after the `...`, so let's put it there